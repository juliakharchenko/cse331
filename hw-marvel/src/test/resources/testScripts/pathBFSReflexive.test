# Tests finding a path from a node to itself (reflexive case).

LoadGraph nba nbaPlayers.csv

FindPath nba DWYANE-WADE DWYANE-WADE