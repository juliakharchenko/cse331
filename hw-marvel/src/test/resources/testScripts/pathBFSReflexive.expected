# Tests finding a path from a node to itself (reflexive case).

loaded graph nba

path from DWYANE-WADE to DWYANE-WADE: